zabbix-agent.sls

```diff
@@ -0,0 +1,31 @@
+zabbix-agent:
+  '4.2.3.2400':
```

Comment:
if you want to make the state more beautiful, you can create multiple variable.
e.g. for the version or installer path, you can look at other states (7zip.sls as an example)